anvil/src/eth/api.rs

```diff
@@ -292,7 +292,7 @@ impl EthApi {
             EthRequest::EvmRevert(id) => self.evm_revert(id).await.to_rpc_result(),
             EthRequest::EvmIncreaseTime(time) => self.evm_increase_time(time).await.to_rpc_result(),
             EthRequest::EvmSetNextBlockTimeStamp(time) => {
-                self.evm_set_next_block_timestamp(time).to_rpc_result()
+                self.evm_set_next_block_timestamp(time.as_u64()).to_rpc_result()
```

Comment:
aren't we downcasting here and possibly overflowing u64?

Reply:
yeh right, not totally safe, will make try_into